test-suite/src/aggregate.rs

```diff
@@ -75,6 +75,11 @@ test_case!(aggregate, async move {
     }
 
     let error_cases = vec![
+        (
+            // once PR #561 is merged this should not produce an error anymore
+            AggregateError::OnlyIdentifierAllowed.into(),
+            "SELECT SUM(ifnull(age, 0)) from Item;",
+        ),
```

Comment:
It is expected to merge this PR earlier than #561 
How about adding this test to #561 rather than here?

Reply:
Ok, let's make this discussion simple.
Adding this test to this is acceptable, but the comment does not look necessary.
> // once PR #561 is merged this should not produce an error anymore

Let's leave this test case but removing the comment.
In #561 you will not be able to miss this comment because the integration test will fail if this PR is merged earlier than #561

Reply:
sounds good to me! ;)